dispatcher/dispatcher.go

```diff
@@ -1087,6 +1088,17 @@ func (d *Dispatcher) isWatching(nodeid NodeId) bool {
 	return ok
 }
 
+func (d *Dispatcher) GetWatchingNodes() []NodeId {
+	wn := make([]NodeId, len(d.watchingNodes), len(d.watchingNodes))
```

Comment:
```suggestion
	watchingNodeIds := make([]NodeId, len(d.watchingNodes), len(d.watchingNodes))
```
Use more meaningful variable names?

Reply:
Agree, changed in the latest commit f1598b6de00e9eae0a5464498d38995f851cb068